data/json/npcs/refugee_center/surface_refugees/NPC_Aleesha_Seward.json

```diff
@@ -151,14 +165,5 @@
       { "text": "What were you saying before?", "topic": "TALK_NONE" },
       { "text": "I'd better get going.", "topic": "TALK_DONE" }
     ]
-  },
-  {
-    "type": "talk_topic",
-    "id": "TALK_REFUGEE_Aleesha_Situation",
-    "dynamic_line": "I don't have much of a hot take besides what I just said.  It's shit.  Not enough to do to stay busy, not enough to eat, and even if I had internet, all the YouTubers I follow are dead.",
```

Comment:
![image](https://user-images.githubusercontent.com/51099123/98988072-169e6980-24f5-11eb-9a1f-fdfa923f8af8.png)
Is there meant to be a replacement line in this PR?

Reply:
Nope, that was an accidental delete. thanks